extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -638,6 +638,66 @@ describe('amp-analytics.visibility', () => {
       });
     });
 
+    it('"hidden" trigger should work with duration condition', () => {
```

Comment:
I'm not sure how exactly a spec like that should behave.

Does that mean "fires when page goes to background and the element has been visible"?
If so, I think a "visible" trigger should be used instead.

Or does that mean "fires when page goes to background and element is currently visible"? Why does people want to track that?